src/client/context.rs

```diff
@@ -1359,6 +1474,10 @@ impl Context {
             .set_presence(game, status, afk)
     }
 
+    /// Remove an undefined amount of members from the given server
+    /// based on the amount of days they've been offline for.
+    ///
+    /// **Note**: This might trigger some amount of [`GuildMemberRemove`] events.
```

Comment:
"This will trigger [\`GuildMemberRemove\`] events"

and let's link to that event by putting this at the bottom:
```rs
/// [`GuildMemberRemove`]: ../model/event/enum.Event.html#variant.GuildMemberRemove
```